src/main/java/RebornStorage/tiles/TileMultiCrafter.java

```diff
@@ -181,51 +152,49 @@ public NBTTagCompound getUpdateTag() {
 	public TileMultiCrafter() {
 	}
 
-	@Nonnull
-	@Override
-	public INetworkNode getNode() {
-		if (!world.isRemote) {
-			INetworkNodeManager manager = API.instance().getNetworkNodeManager(this.world);
-			NetworkNodeMultiCrafter node = (NetworkNodeMultiCrafter) manager.getNode(this.pos);
-			if (node == null || !node.getId().equals("rebornstorage.multicrafter")) {
-				manager.setNode(this.pos, node = getNewNode());
-				manager.markForSaving();
-			}
-			serverNode = node;
-			checkNodes(false);
-			return node;
-		}
+	ItemStack stack = ItemStack.EMPTY;
 
-		if (clientNode == null) {
-			clientNode = getNewNode();
+	public ItemStack getStack() {
+		if(stack.isEmpty()){
+			stack = new ItemStack(ModBlocks.BLOCK_MULTI_CRAFTER, 1, ModBlocks.BLOCK_MULTI_CRAFTER.getMetaFromState(world.getBlockState(getPos())));
 		}
-		checkNodes(false);
-		return clientNode;
+		return stack;
 	}
 
-	public NetworkNodeMultiCrafter getNewNode() {
-		return new NetworkNodeMultiCrafter(this);
+	//RS API
+
+	CraftingNode node;
```

Comment:
No! Don't store this like that!! You're supposed to use the registry!!

``
        INetworkNodeManager manager = API.instance().getNetworkNodeManager(world);

        NetworkNode node = (NetworkNode) manager.getNode(pos);

        if (node == null || !node.getId().equals(getNodeId())) {
            manager.setNode(pos, node = createNode(world, pos));
            manager.markForSaving();
        }

        return (N) node;
``